client/reader/stream/index.jsx

```diff
@@ -458,6 +549,16 @@ class ReaderStream extends Component {
 	}
 }
 
+/* eslint-disable */
```

Comment:
Can we disable specific rules only? Disabling all ESLint rules, even if it's for a few lines, is highly discouraged.